package.json

```diff
@@ -33,5 +33,8 @@
 		"vite": "^4.0.0",
 		"vitest": "^0.25.3"
 	},
-	"type": "module"
+	"type": "module",
+	"dependencies": {
+		"spatext": "^0.3.0-beta.20230312"
```

Comment:
I installed the `spatext` Node package that I just published to https://www.npmjs.com/package/spatext